### PR TITLE
Add variables and generic information handling mechanisms

### PR DESCRIPTION
**Pull request by mklkun**

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -996,6 +996,17 @@ One of the most important notions in workflows is the dependencies between tasks
 print('Hello world')
 ```
 
+===== Variables
+
+To specify https://doc.activeeon.com/latest/user/ProActiveUserGuide.html#_task_variables[task variables],
+you should provide the `variables` parameter. Its value should be a list of tuples `(key,value)` that corresponds to
```
Comment (mcompastie) [inline src/docs/MLOS/MLOSUserGuide.adoc:1002]: you have to provide (not should)

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -1295,15 +1306,12 @@ To delete a task from the workflow, the user should run the pragma `#%delete_tas
 
 ==== Create a job
 
-To create a job, use the `#%job()` pragma:
+To create a job, specify job variables and/or job generic information, use the `#%job()` pragma:
```
Comment (mcompastie) [inline src/docs/MLOS/MLOSUserGuide.adoc:1309]: `specify job variables and/or job generic information by using the `#%job()` pragma:` ?

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 1 above)
Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc:1002]: I prefer to keep "should" to show that this parameter is optional.

File: src/docs/MLOS/MLOSUserGuide.adoc (hunk 2 above)
Comment (mklkun) [reply src/docs/MLOS/MLOSUserGuide.adoc:1309]: Not with a sentence starting by "To create" ;)

**Pull request merged**